Fabric.Authorization.Domain/Services/GroupService.cs

```diff
@@ -32,22 +32,28 @@ public class GroupService
         private readonly IUserStore _userStore;
         private readonly IGroupStore _groupStore;
         private readonly RoleService _roleService;
+        private readonly EDWAdminRoleSyncService _roleManager;
 
         public GroupService(
             IGroupStore groupStore,
             IRoleStore roleStore,
             IUserStore userStore,
-            RoleService roleService)
+            RoleService roleService,
+            EDWAdminRoleSyncService roleManager)
         {
             _roleStore = roleStore;
             _userStore = userStore;
             _groupStore = groupStore ?? throw new ArgumentNullException(nameof(groupStore));          
             _roleService = roleService ?? throw new ArgumentNullException(nameof(roleService));
+            _roleManager = roleManager ?? throw new ArgumentNullException(nameof(roleManager));
```

Comment:
I know we renamed the class, should we also rename the private variable for clarity?